lms/djangoapps/bookmarks/models.py

```diff
@@ -19,16 +51,23 @@ class Bookmark(TimeStampedModel):
     user = models.ForeignKey(User, db_index=True)
     course_key = CourseKeyField(max_length=255, db_index=True)
     usage_key = LocationKeyField(max_length=255, db_index=True)
-    display_name = models.CharField(max_length=255, default='', help_text='Display name of block')
-    path = JSONField(help_text='Path in course tree to the block')
+    _path = JSONField(db_column='path', help_text='Path in course tree to the block')
+
+    xblock_cache = models.ForeignKey('bookmarks.XBlockCache', null=True)
```

Comment:
I don't understand why we are using ForeignKey relation here. My understanding is, there will be a single xblock_cache object against any Bookmark so why not OneToOne, correct me if i'm wrong.